klaviyo/api_helper.py

```diff
@@ -143,6 +166,19 @@ def _is_valid_request_option(self, request_type=PRIVATE):
         if request_type == self.PRIVATE and not self.private_token:
             raise KlaviyoConfigurationException('Private token is not defined')
 
+    def _is_valid_public_method(self, method):
+        """Making sure the HTTP method is valid for public request.
+
+        Args:
+            method (str): method type.
+
+        Raises:
+            (KlaviyoConfigurationException): Information as to why the request won't work.
+        """
+        if method not in [self.HTTP_GET, self.HTTP_POST]:
```

Comment:
nit: this probably shouldn't be mutable:
```suggestion
        if method not in (self.HTTP_GET, self.HTTP_POST):
```